File: unit6/sentimental/cash/cash.py
from cs50 import get_float

# check input more then 0 and number
while True:
    dollar = get_float("Change owed: ")
    if dollar > 0:
        break

# convert to cent
cent = dollar * 100

# count coins
coin25 = int(cent / 25)
coin10 = int((cent - coin25 * 25) / 10)
coin5 = int((cent - (coin25 * 25 + coin10 * 10)) / 5)
coin1 = int((cent - (coin25 * 25 + coin10 * 10 + coin5 * 5)) / 1)

print(coin25 + coin10 + coin5 + coin1)

File: unit1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    //initial variable
    float dollar = 0.0;
    int count = 0;
    int cent = 0;

    // Input data
    do
    {
        printf("Change owed: ");
        dollar = get_float();
    }
    while (dollar <= 0);

    //Convert to cent with round
    cent = roundf(dollar * 100.00);

    //Count coins
    while (cent > 0)
    {
        if (cent >= 25)
        {
            cent -= 25;
        }
        else if (cent >= 10)
        {
            cent -= 10;
        }
        else if (cent >= 5)
        {
            cent -= 5;
        }
        else if (cent >= 1)
        {
            cent -= 1;
        }
        count++;
    }
    //print count
    printf("%i\n", count);
}